### PR TITLE
Feature/sign up

### PR DESCRIPTION
**Pull request by satyam-saxen**

### CODE REVIEW

File: src/components/signUp/signUpResponse.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const SignUpResponse  = ({response})=>{
+
+    console.log(`Response in other file ${response.status}`);
+    
+    const status = response.status;
+
+    if(status === 200 || status === 201){
+        return(<Homepage></Homepage>);
+    }else if(status === 400){
+        return(<h1>Bad Request</h1>)
```
Comment (chandan911) [inline src/components/signUp/signUpResponse.js]: Render the server error message here.

File: src/components/signUp/signUpResponse.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Homepage from './../homepage/homepage';
+const SignUpResponse  = ({response})=>{
+
+    console.log(`Response in other file ${response.status}`);
```
Comment (chandan911) [inline src/components/signUp/signUpResponse.js]: `console.log` lines should not be committed.

File: src/components/signUp/signUpResponse.js (hunk 1 above)
Comment (satyam-saxen) [reply src/components/signUp/signUpResponse.js]: yeah chandan but as i didn't know which parameters are coming from server with status code so i wrote that message but we can change that sure

File: src/components/signUp/signUpResponse.js (hunk 2 above)
Comment (satyam-saxen) [reply src/components/signUp/signUpResponse.js]: yeah got it

**Pull request closed without merging**